www/index.js

```diff
@@ -28,17 +28,11 @@ import './js/intro.js';
 import './js/main.js';
 import './js/survey/survey.js';
 import './js/survey/input-matcher.js';
-import './js/survey/one-click-button.js';
 import './js/survey/multilabel/infinite_scroll_filters.js';
 import './js/survey/multilabel/trip-confirm-services.js';
 import './js/survey/multilabel/multi-label-ui.js';
```

Comment:
Yes; this PR migrates all UI components. The remaining code in those files is just services, and I'm not in a rush to rewrite those.

Leaving the services in Angular for now doesn't block the progress of anything else because they can be accessed from anywhere.

Reply:
Yeah I figured this out later in the review but forgot to come back and modify this comment.